cvat/apps/engine/task.py

```diff
@@ -636,17 +642,28 @@ def _create_thread(
                 filtered_files.append(f)
             data['server_files'] = filtered_files
 
+    # count and validate uploaded files
+    media = _count_files(data)
```

Comment:
It is old code. But "_count_files" doesn't correspond to what the function does. It splits files by the media type. Let's keep as is for now.